search_api_doc.rst

```diff
@@ -0,0 +1,225 @@
+Search API
+==========
+
+Method::
+   
+   GET http://archive.cnx.org/search?q=
+
+Usage::
+   
+   QUERY [author:AUTHOR] [authorID:ID] [title:TITLE] [subject:SUBJECT] ...
+         [keyword:KEY] [type:TYPE] [language:LANG] [pubYear:YEAR] [sort:SORT]
+
+Brakets denote an optional parameter.
+
+
+Parameters:
+
++---------+---------+---------------------------------+
+| Field   | Type    | Description                     |
++---------+---------+---------------------------------+
+| QUERY   | String  | General search string           |
++---------+---------+---------------------------------+
+| AUTHOR  |         | Search by author contact info.  |
++---------+---------+---------------------------------+
+| ID      |         | Search by a specific author ID  |
```

Comment:
This should be authorID and the case of these should match the case we parse for. (I seem to recall a conversation that happened long long about involving authorId vs authorID. I'd hate to rehash that conversations because someone read this wrong. =P